source/includes/examples/cli-json-example-create-clusters-with-autoscaling.rst

```diff
@@ -0,0 +1,43 @@
+.. code-block::
+   :copyable: true
+
+   {
+       "clusterType": "REPLICASET",
+       "links": [],
+       "name": "CustomerPortalProd",
+       "mongoDBMajorVersion": "8.0",
+       "replicationSpecs": [
+         {
+           "numShards": 1,
```

Comment:
I might miss some context, but maybe this field could be removed?

Reply:
I think this is the API response from CLI